src/zre_msg.c

```diff
@@ -35,7 +35,7 @@
 #endif
 
 #include "../include/zyre.h"
-#include "../include/zre_msg.h"
+#include "./zre_msg.h"
```

Comment:
there's no need for ./, it's the default

Reply:
Agreed, I was about to commit without it but I found out that the previous version had the './'